CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.1.0 - 2020-08-10
+### Fixed
+- Handle the default pagination setting
+
+### Changed
+- New interface based on d3.js
+
```

Comment:
Can you add please the ability to use (or not) the history mode ?

Reply:
done, tks